pandas/core/groupby.py

```diff
@@ -1023,6 +1024,61 @@ def tail(self, n=5):
         tail = obj[in_tail]
         return tail
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply func(self, \*args, \*\*kwargs)
```

Comment:
"Apply func" is not really clear, in the sense this does not discrimate it from `apply`. Can you be a bit more specifc here that the full object is passed and not each group.